src/de/jost_net/JVerein/gui/control/FilterControl.java

```diff
@@ -1218,10 +1267,32 @@ public void handleEvent(Event event)
     }
   }
   
+  static class EigenschaftListener2 implements Listener
```

Comment:
Zusatz 2 aus dem Klassennamen sollte entfernt werden. Kann das ggf. mit EigenschaftenListener gemerged werden? Ansonsten aussagekräftigere Unterscheidung zu EigenschaftenListener in der Benennung deutlich machen.